### PR TITLE
Wip api tests

### PR DESCRIPTION
**Pull request by PapsOu**
### Summary

Adds API codeception tests

### CODE REVIEW

File: bin/var-dump-server [hunk 1]
```diff
@@ -0,0 +1 @@
+../vendor/symfony/var-dumper/Resources/bin/var-dump-server
```
Comment (FanchTheSystem) [inline bin/var-dump-server]: Why include this.

File: etc/confd/templates/env.tmpl [hunk 2]
```diff
@@ -12,10 +12,12 @@ DBNAME={{getv "/postgres/default/dbname"}}
 DBUSER={{getv "/postgres/default/username"}}
 DBPASSWORD={{getv "/postgres/default/password"}}
 
-TEST_SERVER_URL="http://127.0.0.1:8000"
+TEST_SERVER_URL="http://{{getv "/url/addr"}}"
 TEST_URL=""
```
Comment (FanchTheSystem) [inline etc/confd/templates/env.tmpl]: Why keep this var

File: bin/var-dump-server (hunk 1 above)
Comment (PapsOu) [reply bin/var-dump-server]: It's an update from Symfony's var-dumper component

File: etc/confd/templates/env.tmpl (hunk 2 above)
Comment (PapsOu) [reply etc/confd/templates/env.tmpl]: .env vars should be cleaned up (see selenium acceptance tests https://github.com/pia-lab/pialab-back/blob/master/tests/_support/WebGuy.php#L47)

**Pull request merged**